### DISCUSSION TITLE
DataGrid Performance issues

### DISCUSSION THREAD

**Discussion by jsheppard-twilio** [twilio-labs/paste#3369] [General] [answered]
Hi all,

Just wondering if the following PR to improve the performance for the DataGrid component was looked into? or implemented?

https://github.com/twilio-labs/paste/pull/2381

Thanks,
Joe

**Comment by serifluous** [comment, marked as answer]
Hey @jsheppard-twilio , there's still a bit of work left on that PR to get it from POC to release. It's currently in our backlog (https://twilio-productivity.atlassian.net/browse/DS-56) but difficult to prioritize over other customer requests.

That being said, we'd definitely appreciate additional contribution to get it over the line!
